packages/session-replay-browser/test/integration.test.ts

```diff
@@ -100,7 +104,7 @@ describe('module level integration', () => {
       expect(record).not.toHaveBeenCalled();
       expect(update).not.toHaveBeenCalled();
       await runScheduleTimers();
-      expect(fetch).not.toHaveBeenCalled();
+      expect(fetch).not.toHaveBeenCalledWith(SESSION_REPLAY_EU_SERVER_URL);
```

Comment:
I made these changes in a previous version of this PR, when I was fetching the remote config from the SR class itself, so these changes aren't really necessary, but they are more precise than the previous version of the test so I decided to leave them in